cmd/agent/app/integrations.go

```diff
@@ -27,24 +27,21 @@ import (
 )
 
 const (
-	tufConfigFile       = "public-tuf-config.json"
 	reqAgentReleaseFile = "requirements-agent-release.txt"
 	constraintsFile     = "final_constraints.txt"
-	tufPkgPattern       = "datadog(-|_).*"
-	tufIndex            = "https://dd-integrations-core-wheels-build-stable.s3.amazonaws.com/targets/simple/"
+	tufPkgPattern       = "datadog-.*"
 	reqLinePattern      = "%s==(\\d+\\.\\d+\\.\\d+)"
-	yamlFilePattern     = "[\\w_]+\\.yaml.*"
-	disclaimer          = "For your security, only use this to install wheels containing an Agent integration " +
+	// Matches version specifiers defined in https://packaging.python.org/specifications/core-metadata/#requires-dist-multiple-use
+	versionSpecifiersPattern = "([><=!]{1,2})([0-9.]*)"
```

Comment:
Not sure if we want to support the full version specification defined in [PEP440](https://www.python.org/dev/peps/pep-0440/#appendix-b-parsing-version-strings-with-regular-expressions):

Feel free to ignore if our versioning will never contemplate such edge cases.

```
VERSION_PATTERN = r"""
    v?
    (?:
        (?:(?P<epoch>[0-9]+)!)?                           # epoch
        (?P<release>[0-9]+(?:\.[0-9]+)*)                  # release segment
        (?P<pre>                                          # pre-release
            [-_\.]?
            (?P<pre_l>(a|b|c|rc|alpha|beta|pre|preview))
            [-_\.]?
            (?P<pre_n>[0-9]+)?
        )?
        (?P<post>                                         # post release
            (?:-(?P<post_n1>[0-9]+))
            |
            (?:
                [-_\.]?
                (?P<post_l>post|rev|r)
                [-_\.]?
                (?P<post_n2>[0-9]+)?
            )
        )?
        (?P<dev>                                          # dev release
            [-_\.]?
            (?P<dev_l>dev)
            [-_\.]?
            (?P<dev_n>[0-9]+)?
        )?
    )
    (?:\+(?P<local>[a-z0-9]+(?:[-_\.][a-z0-9]+)*))?       # local version
"""
```

They provide a small snippet you could maybe use: 

```python
import re
def is_canonical(version):
    return re.match(r'^([1-9]\d*!)?(0|[1-9]\d*)(\.(0|[1-9]\d*))*((a|b|rc)(0|[1-9]\d*))?(\.post(0|[1-9]\d*))?(\.dev(0|[1-9]\d*))?$', version) is not None
```

Reply:
Yeah we have a plan to extend a bit the versioning of our integrations, but for now we are strictly sticking to `major.minor.fix`. I prefer to keep it that way for now, and change it later as we will have to change other things as well for supporting more version formats.

Reply:
I already do this PEP 440 check in my downloader, so we should be good 👍